code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -627,6 +627,9 @@
 	if(method == INGEST)
 		C.take_organ_damage(min(6*toxpwr, reac_volume * toxpwr))
 		return
+	if(method == INJECT)
+		C.take_organ_damage(1.5 * min(6*toxpwr, reac_volume * toxpwr))
```

Comment:
why is this doing the retarded take_organ_damage proc again

can we get rid of that shit and replace it with AdjustBruteLoss like sane people